cmd/gpu-kubelet-plugin/sharing.go

```diff
@@ -200,6 +201,7 @@ func (m *MpsControlDaemon) Start(ctx context.Context, config *configapi.MpsConfi
 		NodeName:                        m.nodeName,
 		MpsControlDaemonNamespace:       m.namespace,
 		MpsControlDaemonName:            m.name,
+		HelmChartName:                   m.manager.config.flags.chartName,
```

Comment:
This isn't going tow ork because you call the variable AppLabelValue in the template itself.

Reply:
My bad, fixed